release_notes.py

```diff
@@ -372,142 +409,139 @@ def modifiedReleaseNotes(self, cnt):
+    def __init__(self):
+        super(ReputationContent, self).__init__()
+        self.show_secondary_header = False
+
+    def load_data(self, data):
         return json.loads(data)
 
-    def getHeader(self):
+    def get_header(self):
         return "Reputations"
 
-    def addedReleaseNotes(self, cnt):
-        #This should never happen
+    def added_release_notes(self, cnt):
+        # This should never happen
         return ""
 
-    def modifiedReleaseNotes(self, cnt):
+    def modified_release_notes(self, cnt):
         rn = cnt.get("releaseNotes", "")
         if len(rn) == 0:
             return None
         res = ""
-        #Add a comment only if there are release notes
+
         if rn != '-':
-            res += "- " + cnt["releaseNotes"] + "\n"
+            res = add_dot(rn) + "\n"
         return res
 
 
 Content.register(ReputationContent)
 
 
 class IntegrationContent(Content):
-    def loadData(self, data):
+    def load_data(self, data):
         return yaml.safe_load(data)
 
-    def getHeader(self):
+    def get_header(self):
         return "Integrations"
 
-    def addedReleaseNotes(self, cnt):
-        res =  "- " + cnt["name"] + "\n"
-        if cnt.get("description") is not None and len(cnt.get("description")) > 0:
-            res += "-- " + cnt["description"] + "\n"
-        return res
+    def added_release_notes(self, cnt):
+        return release_notes_item(cnt["name"], cnt["description"])
 
-    def modifiedReleaseNotes(self, cnt):
+    def modified_release_notes(self, cnt):
         rn = cnt.get("releaseNotes", "")
         if len(rn) == 0:
             return None
         res = ""
-        #Add a comment only if there are release notes
+
         if rn != '-':
-            res =  "- " + cnt["name"] + "\n"
-            res += "-- " + cnt["releaseNotes"] + "\n"
+            res = release_notes_item(cnt["name"], rn)
         return res
 
 
 Content.register(IntegrationContent)
 
-
-releaseNoteGenerator = {
-    "Integrations": IntegrationContent(),
-    "Scripts": ScriptContent(),
-    "Playbooks": PlaybookContent(),
-    "Reports": ReportContent(),
-    "Dashboards": DashboardContent(),
-    "Widgets": WidgetContent(),
-    "IncidentFields": IncidentFieldContent(),
-    "Layouts": LayoutContent(),
-    "Classifiers": ClassifierContent(),
-    "Misc": ReputationContent()
+release_note_generator = {
+    INTEGRATIONS_DIR: IntegrationContent(),
+    SCRIPTS_DIR: ScriptContent(),
+    PLAYBOOKS_DIR: PlaybookContent(),
+    REPORTS_DIR: ReportContent(),
+    DASHBOARDS_DIR: DashboardContent(),
+    WIDGETS_DIR: WidgetContent(),
+    INCIDENT_FIELDS_DIR: IncidentFieldContent(),
+    LAYOUTS_DIR: LayoutContent(),
+    CLASSIFIERS_DIR: ClassifierContent(),
+    REPUTATIONS_DIR: ReputationContent()
 }
 
 
-def parseChangeList(filePath):
-    with open(filePath, 'r') as f:
+def parse_change_list(file_path):
+    with open(file_path, 'r') as f:
         data = f.read()
         return data.split("\n")
     return []
```

Comment:
unreachable code: if success, the first return will execute. otherwise, an exception will be raised